examples/custom/variables.tf

```diff
@@ -17,11 +17,27 @@ variable "shell_settings_enabled" {
 }
 
 variable "mfa" {
-  default     = "NONE"
+  default     = "TOTP4ALL"
   type        = string
   description = "Specify Multi-Factor Authentication method in the account"
 }
 
+variable "user_mfa" {
+  type = set(object({
+    iam_id = string
+    mfa    = string
+  }))
+  default = [{
+    iam_id = "IBMid-3x000xx3xH"
+    mfa    = "LEVEL3"
+    },
+    {
+      iam_id = "IBMid-50xG4CxSQx"
```

Comment:
what are these IAM IDs? We should not include any IDs in the example. We can probably pass them in the pr_test.go if we want to test the functionality and default this variable to an empty list